GeneratorInterface/GenFilters/src/VBFGenJetFilter.cc

```diff
@@ -0,0 +1,104 @@
+#include "GeneratorInterface/GenFilters/interface/VBFGenJetFilter.h"
+
+#include "DataFormats/Math/interface/deltaPhi.h"
+
+#include <HepMC/GenVertex.h>
+
+// ROOT includes
+#include "TMath.h"
+
```

Comment:
There was a usage of "sqrt" before but not anymore. This include can be removed. @davidlange6